validator_instance.go

```diff
@@ -222,6 +222,17 @@ func (v *Validate) RegisterValidationCtx(tag string, fn FuncCtx, callValidationE
 	return v.registerValidation(tag, fn, false, nilCheckable)
 }
 
+// RegisterDurationType is used to register a custom time.Duration type in your
+// application such as the following example. Reasons for implementing a custom
+// time.Duration would be for application specific methods. Note that the
+// validator will handle an aliased time.Duration without the use of this
+// method.
+//
+// Example: type MyDuration time.Duration
+func (v *Validate) RegisterDurationType(t any) {
+	GlobalTimeDurationTypes = append(GlobalTimeDurationTypes, reflect.TypeOf(t))
```

Comment:
I do not think this should mutate a global state. This should probably belong to the specific validator instance.

Reply:
Oh good catch, I hadn't looked closely enough to realize that the other RegisterFoo functions were operating on their `*Validate` receiver. Agreed.